recipes/helanal/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "helanal" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/helanal-{{ version }}.tar.gz
+  sha256: b53370d13e5d22d8179c5da9e193af2aa853851e89611241500a906434a7616c
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python >=3.9
+    - setuptools >=61.2
+    - versioningit
+    - pip
+  run:
+    - python >=3.9
+    - mdanalysis >=2.0.0
```

Comment:
```suggestion
    - mdanalysis >=2.0.0
    - gsd <3.4
```

Reply:
kk, thanks! I had a similar thought but wasn't sure if there was a 'nicer' way to resolve it